packages/app-root/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@zooniverse/app-root",
+  "description": "",
+  "license": "Apache-2.0",
+  "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "type": "module",
```

Comment:
Heads up that `next-i18next` doesn't support modules. 😒

https://github.com/i18next/next-i18next/issues/2185